website/docs/data-display/d3-chart/examples/low-level-components-use.tsx

```diff
@@ -7,10 +7,12 @@ const Demo = () => {
   const patternSymbolSize = getPatternSymbolSize({ patternKey, patterns });
 
   return (
-    <svg height='100px' width='200px'>
+    <svg height='100px' width='200px' aria-label='PatternFill and PatternSymbol' role='img'>
       <PatternFill id='pattern-element' patternKey={patternKey} color='red' patterns={patterns} />
       <rect width='100px' height='100px' x='0' y='0' fill='url(#pattern-element)' stroke='red' />
       <PatternSymbol
+        aria-label='zigzag PatternSymbol'
+        role='img'
```

Comment:
is it ok to use `img` in `img`?

Reply:
good question, but it seems that it's ok, Axe doesn't complain, and even MDN has nested images in examples https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role
I guess we'll see the final answer in the next LA recheck

Reply:
gave it a bit of thought and changed role to "none" for the nested svg
the standard says img's children are supposed to be presentational
and it fixed the double reading in NVDA